packages/lodestar/src/network/gossip/validation/block.ts

```diff
@@ -90,6 +94,16 @@ export async function validateGossipBlock(
     });
     return ExtendedValidatorResult.reject;
   }
+  if (
+    toHexString(await chain.forkChoice.getAncestor(blockRoot, finalizedSlot)) !== toHexString(finalizedCheckpoint.root)
```

Comment:
config.types.Root.equals()

Reply:
also you aren't accounting for when getAncestor returns undefined. YOu could make generic method which works here and in attestation validation

Reply:
going to use isDescendantOfFinalized for both